app/src/main/java/de/danoeh/antennapod/fragment/DiscoveryFragment.java

```diff
@@ -137,7 +145,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         butRetry = root.findViewById(R.id.butRetry);
         txtvEmpty = root.findViewById(android.R.id.empty);
 
-        loadToplist(countryCode);
+        loadUserSubscriptions();
```

Comment:
I would actually not remove them from the "more" screen. Maybe mark them as subscribed, but it somehow feels strange to not have a place where all suggestions are shown. Then you don't have a feeling for what is trending, if half of the list is missing. What do you think @keunes?

Reply:
Sorry for the late reply. The user at that point is looking for new podcasts, not to learn the top 10 at that point in that country. I would therefore hide the ones they are already subscribed to, if this can be done in a neat way.